sound/soc/intel/boards/sof_sdw.c

```diff
@@ -821,11 +821,11 @@ static int sof_card_dai_links_create(struct device *dev,
 /* SoC card */
 static char components_string[] = "cfg-spk:2"; /* cfg-spk:%d */
 #if !IS_ENABLED(CONFIG_SND_SOC_INTEL_USER_FRIENDLY_LONG_NAMES)
-/* Can also be sof-sdw-mono */
-static char sdw_card_long_name[] = "sof-sdw-stereo";
+/* Can also be sof-soundwire-mono */
+static char sdw_card_long_name[] = "sof-soundwire-stereo";
```

Comment:
That's a good point. SDW nicely sits next with HDA, but if it's not used elsewhere, then it will be very misleading.

Reply:
sorry, @kv2019i i could not understand your comment.
in general HDaudio codecs and SoundWire are mutually exclusive due to pin-muxing. Not sure what you meant by 'nicely sits next with HDA'?

Also we need need to fix the card name, it's not stereo or mono but 2ch or 4ch. we confused mono and single amp.

Reply:
@plbossart 
> sorry, @kv2019i i could not understand your comment.
> in general HDaudio codecs and SoundWire are mutually exclusive due to pin-muxing. 

Aa, sorry, I'm fine with the change. I was purely thinking about the naming. We tend to like our three letter acronyms (like we use HDA instead of HD-Audio), so in that sense "SDW" would be the natural term for SoundWire.  But yeah, probably rest of the world would prefer more letters, so ++votes for sof-soundwire-stereo.